lib/numberpicker.dart

```diff
@@ -116,13 +121,23 @@ class NumberPicker extends StatelessWidget {
   /// if min=0, max=5, step=3, then items will be 0 and 3.
   final int step;
 
+  ///Repeat values infinitely
+  final bool infiniteLoop;
+
+  ///Amount of items
+  int itemCount;
```

Comment:
StatelessWidget should not have mutable field. Could you move it back as a variable in method?